configs/apollodata.json

```diff
@@ -4,17 +4,23 @@
     "https://www.apollographql.com/docs/"
   ],
   "stop_urls": [
-    "index\\.html"
+    "#"
   ],
   "selectors": {
-    "lvl0": ".content h1",
-    "lvl1": ".content h2",
-    "lvl2": ".content h3",
-    "lvl3": ".content h4",
-    "lvl4": ".content h5",
-    "text": ".content p, .content li"
+    "default": {
+      "lvl0": {
+        "selector": "div.sidebar span.title-sidebar",
+        "default_value": "Documentation"
+      },
+      "lvl1": ".header-wrapper h1",
+      "lvl2": ".content-wrapper h2",
+      "lvl3": ".content-wrapper h3",
+      "lvl4": ".content-wrapper h4",
+      "text": ".content-wrapper p, .content-wrapper li, .header-wrapper div.subtitle-page"
+    }
   },
-  "min_indexed_level": 2,
+  "only_content_level": true,
```

Comment:
Similar to my previous comment, I'm not sure if this is necessary and can't seem to find the documentation for it, but this might not be necessary (or might be covered by `min_indexed_level` below?)  Some explanation of what this does would be appreciated!

Reply:
Documentation regarding this parameter are available here https://community.algolia.com/docsearch/documentation/docsearch-scraper/config-options/#only_content_level-optional though

If you use this parameter, you don't need to set `min_indexed_level`

Reply:
Does `only_content_level` === `min_indexed_level: 1` (specifically "1")?

Reply:
Nope, it override completely the behaviour of it. It tells the crawler to only populate to algolia records built with a content not empty. This content is the text matched by the `text` selector